assets/src/edit-story/elements/media/index.js

```diff
@@ -54,7 +54,6 @@ export const MEDIA_DEFAULT_ATTRIBUTES = {
   scale: 100,
   focalX: 50,
   focalY: 50,
-  isFill: false,
```

Comment:
Also need to remove `canFill` in this file and others?

Reply:
Thanks! Completely missed that. Removed now.